File: Exercises_C++/1164.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

	int qntd;
	cin >> qntd;

	for (int c = 0 ; c < qntd ; c++) {

		int soma, numero;
		cin >> numero;
		soma = 0;

		for (int i = 1 ; i < numero ; i++) {
			
			if (numero % i == 0) {
				soma += i;
				}
		

			if (soma > numero) {
				break;
				}

		}

		if (soma == numero) {
                cout << numero << " eh perfeito" << endl;
            }
            else {
                cout << numero << " nao eh perfeito" << endl;
            }
		
	}
		
}

File: Bee-Crowd/Exercises_Python/1164.py
numero = int(input())

for count in range(0,numero):
    n = int(input())
    soma = []
    for i in range(1,n):
        if n % i == 0:
            soma.append(i)
    if sum(soma) == n:
        print('{} eh perfeito'.format(n))
    else:
        print('{} nao eh perfeito'.format(n))